models/articles.models.js

```diff
@@ -48,3 +62,35 @@ exports.fetchCommentsByArticleId = (article_id) => {
     return comments;
   });
 };
+
+exports.insertCommentByArticleId = (article_id, comment) => {
+  // Check for properties
+  const requiredProperties = ["username", "body"];
+  const missingProperties = requiredProperties.filter(
+    (property) => !(property in comment)
+  );
+  if (missingProperties.length) {
+    return Promise.reject({
+      status: 400,
+      msg: `Missing required properties: ${missingProperties.join(", ")}`,
+    });
+  }
```

Comment:
I don't think you'll need all these checks, pretty sure PSQL should be throwing an error if this is the case - after all, in our DB, we have "not null" constraints:
`        body VARCHAR NOT NULL,
        article_id INT REFERENCES articles(article_id) NOT NULL,
        author VARCHAR REFERENCES users(username) NOT NULL,`